osscameroon/camerdevs
# feat: add city filter on Get Ratings

This pr add add city filter on Get Ratings. Related to #275

Steps to verify:

  - move to the backend folder with cd ./backend
  - run make start-postgres to run an initialized database
  - run make run to launch the api
  - then run the following commands

1️⃣ Get ratings of a city
```shell 
curl -X GET 'http://localhost:7000/ratings?page=1&limit=10&city=Maroua'
```

you should see every entry with city equal to Maroua
